src/components/forms/AddHouseholdForm.js

```diff
@@ -33,67 +33,58 @@ function AddHouseholdForm({ visible, onCreate, onCancel }) {
           }}
         >
           <Form.Item
-            label="Address"
-            name="address"
+            label="location Id"
+            name="location_id"
             rules={[
               {
                 required: true,
-                message: 'Please enter the address',
+                message: 'Please enter the location id',
               },
             ]}
           >
-            <Input placeholder="Enter Street Address" size="large" />
+            <Input placeholder="Enter location id" size="large" />
           </Form.Item>
           <Form.Item
-            label="City"
-            name="city"
+            label="Household Name"
```

Comment:
I'm wondering if instead of adding "household name" we could possibly call this "Home Address" and this is where we type out the full address. "household name is too obscure and doesn't make a whole lot of sense to me, or especially to the user.

Reply:
Also, I would have liked to have seen your final commit, currently the only commits I see are about lines 20 and 21

Reply:
Household name is intended to be a way to identify a family group of recipients. An address would not be appropriate here because that can change (a bunch of times or potentially not exist at all for a period of tim) for a given household.

Reply:
@Anthony-Cortese Yeah, I didn't think my first commit went thru, so I copied it into my 2nd commit, but you can see the different info at the end of the 2nd commit. I'll pay closer attention next time.